index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <section class="header-section">
+        <div class="header-container">
+          <div class="title">
+            <img class="logo" src="image/logo.png" alt="logo" />
+            <a href="/">
+              <img class="pandamarket-logo" src="image/pandamarket.png" alt="pandamarket" />
+            </a>
+          </div>
```

Comment:
제 생각에 아래와 같이 마크업하는데 더 간결하다는 생각이 드네요. 

로고를 눌러도 `/` 페이지로 이동을 해야합니다. 

```suggestion
          <a class="title">
            <img class="logo" src="image/logo.png" alt="logo" />
             <img class="pandamarket-logo" src="image/pandamarket.png" alt="pandamarket" />                    </a>
```